editor/js/Sidebar.Scene.js

```diff
@@ -192,6 +192,9 @@ function SidebarScene( editor ) {
 	const backgroundBlurriness = new UINumber( 0 ).setWidth( '40px' ).setRange( 0, 1 ).onChange( onBackgroundChanged );
 	backgroundEquirectRow.add( backgroundBlurriness );
 
+	const backgroundIntensity = new UINumber( 1 ).setWidth( '40px' ).setRange( 0, 10 ).onChange( onBackgroundChanged );
```

Comment:
@WestLangley What is your recommendation for the value range here? What maximum intensity value would still be sensible from your point of view?

Reply:
Since there is no slider, I think setting the max to `Infinity` is best. Plus, it avoids the issue. :-)